test/jdk/javax/swing/Popup/TaskbarPositionTest.java

```
@@ -209,25 +214,38 @@ private void maybeShowPopup(MouseEvent e) {
         }
     }
 
+    // for debugging purpose, saves screen capture when test fails.
+    private static void saveScreenCapture(Rectangle bounds) {
+        BufferedImage image = robot.createScreenCapture(bounds);
+        try {
+            ImageIO.write(image,"png", new File("Screenshot.png"));
+        } catch (IOException e) {
+            e.printStackTrace();
```

I'm not sure it's the best idea to hide the error and just print it into stdout. Wouldn't it be better to just throw the error, what do you think?

---

Yes, we don't care about `IOException`, it can occur only after the test already failed, and it's important to preserve the original exception.